RELEASE.md

```diff
@@ -1,52 +1,54 @@
-# Release instructions
+# Release Instructions
 
 Before creating a new release please do a careful consideration about the
```

Comment:
Before creating a new release carefully consider the version number of the new release.